Classes/Register.php

```diff
@@ -111,16 +111,21 @@ public static function createTcaConfiguration(array $configuration): void
             ],
         ];
 
+        if($GLOBALS['TCA'][$tableName]['columns']['calendarize_info']['config']['parameters']['items']){
+            $items = $GLOBALS['TCA'][$tableName]['columns']['calendarize_info']['config']['parameters']['items'];
+        }
+        
         $GLOBALS['TCA'][$tableName]['columns']['calendarize_info'] = [
             'label' => 'LLL:EXT:calendarize/Resources/Private/Language/locallang.xlf:tca.information',
             'config' => [
                 'type' => 'none',
                 'renderType' => 'calendarizeInfoElement',
                 'parameters' => [
-                    'items' => 10,
+                    'items' => $items ?? 10,
```

Comment:
Perhaps we should change this direclty to 
```'items' => $GLOBALS['TCA'][$tableName]['columns']['calendarize_info']['config']['parameters']['items'] ?? 10,           ```
and drop the separated $items variable?